CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## [v1.9.0-dev] - Unreleased
```

Comment:
Can you add `## [v1.10.0-dev] - Unreleased`

Reply:
I'll create a separate commit/PR for this once the version is released ;)